src/classes/ALLO_Allocations_TDTM.cls

```diff
@@ -434,20 +510,23 @@ public class ALLO_Allocations_TDTM extends TDTM_Runnable {
     * @param listOpps A list of opportunities.
```

Comment:
Super minor, but let's update this param line now that SObjects are allowed. Could list the objects that this is intended for?